JenkinsfileStressTests.groovy

```diff
@@ -0,0 +1,100 @@
+// This file should respect the Jenkinsfile format describe here: 
+// https://jenkins.io/doc/book/pipeline/jenkinsfile/
+// 
+// It may be used by any Jenkins instance you own.
+//
+// In order to work properly, it requires the following parameters:
+// - SENARIO: the Gatling scenrio you want to play (ex. com.linagora.gatling.imap.scenario.ImapSimpleScenario)
+
+// Method in order to retry a command.
+// It may be used to wait for a service to be available.
+
+pipeline {
+    agent none
+
+    stages {
+        stage('Prepare target') {
+            agent {
+                node {
+                    label 'target'
+                }
+            }
+
+            tools {
+                maven 'maven'
+            }
+
+            stages {
+                stage('Compile') {
+                    steps {
+                        sh "mvn clean install -T1C -DskipTests"
+                    }
+                }
+                stage('Build image') {
+                    steps {
+                        script {
+                            sh "cp server/container/guice/cassandra-rabbitmq-guice/target/james-server-cassandra-rabbitmq-guice.jar dockerfiles/run/guice/cassandra-rabbitmq/destination"
+                            sh "cp -r server/container/guice/cassandra-rabbitmq-guice/target/james-server-cassandra-rabbitmq-guice.lib dockerfiles/run/guice/cassandra-rabbitmq/destination"
+                            sh "cp server/container/cli/target/james-server-cli.jar dockerfiles/run/guice/cassandra-rabbitmq/destination"
+                            sh "cp -r server/container/cli/target/james-server-cli.lib dockerfiles/run/guice/cassandra-rabbitmq/destination"
+                            sh 'cp server/protocols/jmap-integration-testing/rabbitmq-jmap-integration-testing/src/test/resources/keystore dockerfiles/run/guice/cassandra-rabbitmq/destination/conf'
+                            sh 'docker build -t james_run dockerfiles/run/guice/cassandra-rabbitmq'
+                        }
+                    }
+                }
+                stage('Start James') {
+                    steps {
+                        script {
+                            sh "cd /srv && sudo btrfs subvolume snapshot bench-snapshot bench-running-docker"
+                            sh 'docker run -d --name=cassandra -p 9042:9042 -p 9160:9160 -v /srv/bench-running-docker/cassandra:/var/lib/cassandra cassandra:3.11.3'
+                            sh 'docker run -d --name=elasticsearch -p 9200:9200 -p 9300:9300 -v /srv/bench-running-docker/elasticsearch:/usr/share/elasticsearch/data/elasticsearch  --env "discovery.type=single-node" elasticsearch:6.7.2'
```

Comment:
you dont need port 9300

Reply:
FYI you have a downgrade of ES to 6.3.2 coming :trollface:

Reply:
I'm afraid we can't, since provisioning have been done on 6.7.2

Reply:
we can: just need to reindex
btw you did not remove port 9300 binding?